### PR TITLE
carrots demo

### PR DESCRIPTION
**Pull request by codeboy5**
## Description

This PR partially adds functions for the completing the carrots demo.

Partial issue #59  

## Type of change

Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Did you run `make test-all`? Yes I ran `make test`

## Checklist:

- [ ] New Unit tests added
- [ ] Unit tests pass locally with my changes

### CODE REVIEW

File: examples/carrots.py [hunk 1]
```diff
@@ -51,16 +51,12 @@ def private_mean(self, privacy_budget: float) -> dp.StatusOrO:
...
-        pass
-
-    def private_count_above(self, privacy_budget: float, limit: int) -> dp.StatusOrO:
-        pass
+        x = dp.CountInt(privacy_budget)
+        return x.result(list(self._df[self._df.carrots_eaten > limit]["carrots_eaten"]))
 
     def private_max(self, privacy_budget: float) -> dp.StatusOrO:
-        pass
+        x = dp.Max(privacy_budget, 0, 150)
```
Comment (chinmayshah99) [inline examples/carrots.py:63]: Please add a comment on what 0 and 150 represent

Comment (codeboy5) [reply examples/carrots.py:63]: They are the search bounds, default values are numeric integers limits which gives a very inconsistent value.
Should I add a comment saying these are the search bounds ?

Comment (chinmayshah99) [reply examples/carrots.py:63]: Yes, please do that. 
Also, can you please add it in other part of code if it's not present?

Comment (codeboy5) [reply examples/carrots.py:63]: Okay, will do that

Comment (codeboy5) [reply examples/carrots.py:63]: I've added the comment, the private sum requires similar bounds but we haven't ported that yet.

**Review by chinmayshah99 — APPROVED** [review]

**Pull request merged**